A3A/addons/core/functions/UtilityItems/fn_initObject.sqf

```diff
@@ -20,6 +20,11 @@ Example:
 
 params [["_object", objNull, [objNull]]];
 
+_object setVehicleRadar 0; ///not sure if all of that, will see
+_object setVehicleReceiveRemoteTargets true;
+_object setVehicleReportRemoteTargets true;
+_object setVehicleReportOwnPosition true;
```

Comment:
Is this actually needed?

Reply:
Ah, not sure about initObject, I might rewrite it so it checks for vehicles that are considered SAM or something like that, now it makes even MGs translate their position on the radar, which can be distracting